packages/access-client/src/types.ts

```diff
@@ -60,6 +60,13 @@ export interface SpaceTableMetadata {
   agent: AgentMeta
 }
 
+/**
+ * Indicates failure executing ability that requires access to a space that cannot be found by the handler
+ */
+export type SpaceNotFoundFailure = Failure & {
+  status: 404
```

Comment:
```suggestion
  name: 'UnknownSpaceDID`
```

I would say ☝️, which is I think more accurate than not found. If we want to be future compatible I would go as far as do this instead.

```suggestion
  name: 'NoStorageProvider'
```

Reply:
For what it's worth I don't mind `status` along with the `name` to categorize concrete errors into a groups with HTTP semantics, I just think it's not replacement for concrete errors.

Reply:
I got rid of `status` and used `name` of `UnknownSpace` in https://github.com/web3-storage/w3protocol/pull/391/commits/8f0e5e342e858f25eb832bde4be2ea90a07175ae#diff-f4e938a2d743868395a4bb96adeecd10ce306ac53100e1e978df8d40ac33d4ef